File: c/b2292.c
#include <stdio.h>

// 2292: 벌집

int main(void) {
	int N;
	int count = 1;
	int total = 1;

	scanf("%d", &N);

	while (N > total) {
		total += count * 6;
		count++;
	}

	printf("%d", count);
	return 0;
}

File: rust/b2292.rs
use std::io;

// 2292: 벌집

fn main() {
    let mut N = String::new();
    io::stdin().read_line(&mut N).expect("failed: input");
    let N : u32 = N.trim().parse().expect("failed: parse");

	let mut count: u32 = 1;
	let mut total: u32 = 1;

	while N > total {
		total += count * 6;
		count += 1;
	}

	println!("{}", count);
}